library/Booster/Pattern/Base.hs

```diff
@@ -415,37 +418,96 @@ pattern DotDotDot = DomainValue (SortApp "internalDummySort" []) "..."
 {- | A predicate describes constraints on terms. It will always evaluate
    to 'Top' or 'Bottom'. Notice that 'Predicate's don't have a sort.
 -}
```

Comment:
```suggestion
{- | A predicate describes constraints on terms. It will always evaluate
   to 'TrueBool' or 'FalseBool'. Notice that 'Predicate's don't have a sort.
-}
```